app/app.js

```diff
@@ -48,10 +48,10 @@ const SENTRY_DSN =
     ? process.env.REACT_APP_SENTRY_PROD_DSN
     : process.env.REACT_APP_SENTRY_DEV_DSN;
 
-Sentry.init({
-  dsn: SENTRY_DSN,
-  integrations: [new BrowserTracing()],
-});
+// Sentry.init({
```

Comment:
Is Sentry not working?  Or do we not want to call this?

Reply:
I turned it off to prevent unnecessary alerts during development. nice catch 👍🏾